client/src/pages/gptsListPage.tsx

```diff
@@ -1,9 +1,13 @@
 import ListOfUsersGPT from '../components/lists';
 import CreateGPTBtn from '../components/createNewGPTBtn';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import Grid from '@mui/material/Grid';
```

Comment:
I think it can be something like
import {Box, Button, ... } from '@mui/material';